redaxo/src/addons/structure/plugins/history/boot.php

```diff
@@ -57,7 +57,7 @@
 
     rex_extension::register('ART_SLICES_QUERY', static function (rex_extension_point $ep) {
         $history_date = rex_request('rex_history_date', 'string');
-        $history_revision = rex_request('history_revision', 'int', 0);
+        $history_revision = rex_request('history_revision', 'int', intval(rex::getProperty('login')->getSessionVar('rex_version_article')));
```

Comment:
ist das intval dafür da, damit hier 0 rauskommt wenn die session var nicht gesetzt ist?

Fallls ja, wäre ggf. Sowas eindeutiger (an allen stellen)
```suggestion
        $history_revision = rex_request('history_revision', 'int', rex::getProperty('login')->getSessionVar('rex_version_article', 0));
```